package.xml

```diff
@@ -11,16 +11,21 @@
 	<url type="repository">https://github.com/ethz-asl/libnabo</url>
 	<author email="stephane.magnenat@mavt.ethz.ch">Stéphane Magnenat</author>
 
-	<!-- The *_depend tags are used to specify dependencies -->
-	<buildtool_depend>cmake</buildtool_depend>
-	<build_depend>eigen</build_depend>
+	<!-- If a pure-cmake build is desired, the buildtool should be switched to 'cmake' -->
+	<buildtool_depend>catkin</buildtool_depend>
+	
 	<build_depend>boost</build_depend>
-	<run_depend>eigen</run_depend>
+	<build_depend>eigen</build_depend>
+	<build_depend>libopenmp-dev</build_depend>
+
 	<run_depend>boost</run_depend>
-	<run_depend>catkin</run_depend>
+	<run_depend>eigen</run_depend>
```

Comment:
Eigen is a header-only library, so no need to run_depend on it.

Reply:
We also don't need to list boost as run_depend. I'll remove them.

Reply:
Done.